<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportOptionsDialog</class>
 <widget class="QDialog" name="ExportOptionsDialog">
  <property name="windowTitle">
   <string>Export Options</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width> <!-- Adjusted for three 180px buttons + spacing -->
    <height>250</height> <!-- Adjusted for title + buttons + margins -->
   </rect>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>How would you like to export?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size:18px; font-weight:bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="optionsLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QToolButton" name="copyButton">
       <property name="text">
        <string>Export as a copy</string>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="fixedSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <!-- StyleSheet for border will be set dynamically -->
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tendiesButton">
       <property name="text">
        <string>Export as .tendies</string>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="fixedSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <!-- StyleSheet for border will be set dynamically -->
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nuggetButton">
       <property name="text">
        <string>Export to Nugget</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/assets/nugget.png</normaloff>:/assets/nugget.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="fixedSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <!-- StyleSheet for border will be set dynamically -->
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui> 